org_status/org_status.py

```diff
@@ -75,19 +76,41 @@ def get_argument_parser():
     parser.add_argument('--verbose', '-v', action='store_true')
     parser.add_argument('--hosts-only', '-o', action='store_true')
     parser.add_argument('--skip-host-checks', action='store_true')
+    parser.add_argument('--export-orgs', type=str)
+    parser.add_argument('--format', type=str, default='gitman')
 
     return parser
 
 
-def main():
-    parser = get_argument_parser()
-    args = parser.parse_args()
+parser = get_argument_parser()
+args = parser.parse_args()
+
+styled = (lambda l, *_: l) if args.no_color else colored
+verbose = print if args.verbose else (lambda *_: None)
+
+hosts_only_print = print if args.hosts_only else verbose
+
+
+def encode_repo_list(repo_data, exporter_name):
+    exporters = get_all_supported_encoders()
```

Comment:
Please take of the terminology both here and elsewhere. This variable needs to be named as `encoders`.